gordo_components/server/views/base.py

```diff
@@ -215,15 +213,16 @@ class MetaDataView(Resource):
     Serve model / server metadata
     """
 
+    @server_utils.metadata_required
```

Comment:
I must admit that I definitely dont see the prettyness in having these as decorators which puts data inside the magic variable `current_app`, vs just calling a function inside this get.

Reply:
This is, in effect, exactly how we had it before. Server would load it, but serialize the loaded model upon request. But alas, your wish is my command.